src/graphql/error-map.ts

```diff
@@ -185,6 +188,13 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
     case "LocalCacheServiceError":
     case "LocalCacheUndefinedError":
     case "UnknownLocalCacheServiceError":
+    case "UserLoginIpRateLimiterExceededError":
```

Comment:
As all these errors are collapsed I guess we don't need the granularity or? Wouldn't a simple `RateLimitExceeded` be enough?

Reply:
I see a point to know which rate limiter are most frequently applied, specially following potential DDOs, therefore it make sense to differentiate them.